lib/faker.rb

```diff
@@ -0,0 +1,21 @@
+class Faker
+  attr_reader :default_count
+  def initialize(default_count = 5)
```

Comment:
Not a concern atm, but we'll be increasing this as the project grows, right? To a couple hundred possibly.

Reply:
Where this default lives in the code will likely change as we iterate and build relationships with content. The idea is we can set this from the commands we run or set them to defaults we want.

e.g. `rake seed:users NUM=5000`